include/rmm/mr/device/fixed_multisize_memory_resource.hpp

```diff
@@ -201,8 +209,9 @@ class fixed_multisize_memory_resource : public device_memory_resource {
 
   Upstream* upstream_mr_;  // The upstream memory_resource from which to allocate blocks.
 
-  std::size_t min_size_; // size of smallest blocks allocated
-  std::size_t max_size_; // size of largest blocks allocated
+  std::size_t size_base_;         // base of the allocation block sizes (power of 2)
```

Comment:
```suggestion
  std::size_t const size_base_;         // base of the allocation block sizes (power of 2)
```

Reply:
Since these are being used for on the fly exponentiation to compute sizes, there's a better chance the compiler will optimize that out if these are const.